recipes/galeshapley/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "galeshapley" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/galeshapley-{{ version }}.tar.gz
+  sha256: 39e8df83c203ecefcf51b89346b4094f8243dddde98a95581bb0e3c8530dfde2
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
```

Comment:
Since the upstream `setup.py` has a `setup_requires` specifying `pytest-runner`, I believe you will have to include `pytest-runner` under `host`.